plugins/hydra_ax_sweeper/tests/test_ax_sweeper_plugin.py

```diff
@@ -145,7 +145,7 @@ def test_jobs_configured_via_cmd_and_config(hydra_sweep_runner) -> None:
         overrides=[
             "hydra/launcher=basic",
             "hydra.sweeper.ax_config.max_trials=2",
-            "quadratic.x=-5:-2",
```

Comment:
how come all your tests are in integers?

Reply:
Not all. https://github.com/facebookresearch/hydra/pull/912/files#diff-7e2f83750c977fec26bb8f3b6738e2edR178

Reply:
I can add another test or make some of them regular interval